app/retail/templates/emails/cart.html

```diff
@@ -0,0 +1,64 @@
+{% extends 'emails/template.html' %}
+{% comment %}
+  Copyright (C) 2020 Gitcoin Core
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published
+  by the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+{% load i18n humanize %}
+
+{% block content %}
+
+<style>
+  .arrow-up{
+    width: 0;
+    height: 0;
+    border-left: 15px solid transparent;
+    border-right: 15px solid transparent;
+    border-bottom: 15px solid #ddd;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  #founder_announce {
+    background-color: #fafafa;
+    margin-bottom: 15px;
+  }
+
+  #founder_announce a{
+    font-weight: bold;
+  }
+</style>
+
+<div>
+  <h1 class="text-centre">{% trans "Your cart is waiting for you" %}</h1>
+</div>
+
+<a href="https://gitcoin.co/owocki">
+<img style="width: 100px; height: 100px;" src="https://gitcoin.co/dynamic/avatar/owocki">
+</a>
+<div class="arrow-up">&nbsp;</div>
+<div id="founder_announce" style="border: 1px solid #ddd; max-width: 500px; text-align: center; margin-left: auto; margin-right: auto; margin-bottom: 25px;">
+<p style="margin: 5px; border-radius: 5px;">
+  {{desc|safe}}
```

Comment:
can we pls list the grants that are in the users cart + the token amounts associated with them here ppls?